### ISSUE TITLE
Shell user and ros folder placement

### ISSUE THREAD

**Issue by wieset** [esa-prl/rover_docker#3]
Currently the container uses a root shell and `/home/ros` for development. For root, the home folder is actually `/root,` so the correct place would be `/root/ros` to make things like `~/ros/marta_ws` work. However, I think it would actually be better to run a shell with a non-root user, to better emulate our environment, and place the ros folder accordingly.

**Comment by levingerdes** [comment]
I agree, we should try it as a non-root user to catch permission and path issues. In my old Rock Docker images, I prepared this like so:

```
# create and switch to non-root user in sudoers group
run adduser --disabled-password --gecos '' user &&\
    adduser user sudo &&\
    echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
user user

arg ROCK_DIR=/home/user/rock/

run mkdir -p $ROCK_DIR
workdir $ROCK_DIR
```

**Comment by mvoellmy** [comment]
does this appear

**Comment by mvoellmy** [comment]
test

**Comment by maxehrhardt** [comment]
Resolved by commit 9946c0346dc357ed9572f7749dc3cfb1a565d00f
